tests/widget/modal/widget-modal.login.test.js

```diff
@@ -22,3 +22,54 @@ test('modal widget', async ({ page }) => {
   expect(await fullName.innerText()).toBe('Full name: Demo User');
   expect(await email.innerText()).toBe('Email: demo@user.com');
 });
+
+test.only('modal widget accessibly', async ({ page }) => {
+  const loginButton = page.locator('button', { hasText: 'Open Login Modal' });
+  const dialog = page.locator('dialog');
+  expect(await dialog.isVisible()).toBeFalsy();
+
+  await loginButton.click();
+  // Add just a bit of a delay to ensure dialog responds
+  await page.waitForTimeout(1000);
+  expect(await dialog.isVisible()).toBeTruthy();
+  await page.goto('widget/modal');
+  const username = page.getByRole('textbox', { name: /User Name/ });
+  const password = page.getByRole('textbox', { name: /Password/ });
+  const eye = page.getByRole('button', { name: 'show-password' });
```

Comment:
You should be able to find this button with `"Show Password"` and not need the addition of the `aria-label`. Here's a screenshot of the current app with the Accessibility panel open: https://share.getcloudapp.com/7KukDq6P. As you can see, the button should have an accessible name, but if it isn't working reliably, then let me know.